news.d/feature/1333.ui.md

```diff
@@ -0,0 +1 @@
+Show a message when hiding to tray the first time.
```

Comment:
Should this be "the first time for each Plover launch" or something similar?

If it's truly "the first time" Plover should probably store some config option "show message on minimize" with options (I think) "never", "once", "once every launch", "always"